etc/lime-elements.api.md

```diff
@@ -652,6 +652,7 @@ export namespace Components {
         "label"?: string;
         "placeholder"?: string;
         "readonly"?: boolean;
+        "required": boolean;

```

Comment:
Should `required` be optional here as well?

Reply:
Oh yes absolutely. I just forgot that.

Reply:
fixed